pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -99,6 +104,35 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return errors.Wrap(err, "could not create watch on WindowsMachineConfig objects")
 	}
+	// Watch for the Machine objects
+	windowsOSLabel := "machine.openshift.io/os_id"
```

Comment:
I am fine with this. @PratikMahajan are you in agreement?

Reply:
yes, changed that after discussing with @ravisantoshgudimetla. basically, machine-api guys are using `machine.openshift.io` for all their labels. Thought to keep it consistent.

Reply:
but one more thing, 
they're using `-` and not `_`, not sure if we should make it `os-id` or keep it `os_id`

Reply:
We should use `os-id` then